src/rpc/rawtransaction.cpp

```diff
@@ -1247,7 +1247,7 @@ static RPCHelpMan decodepsbt()
             for (const auto& tuple : tuples) {
                 uint8_t depth = std::get<0>(tuple);
                 uint8_t leaf_ver = std::get<1>(tuple);
-                CScript script = std::get<2>(tuple);
+                std::vector<unsigned char> script = std::get<2>(tuple);
```

Comment:
if you retouch: this seems simplier, same above
```suggestion
            for (const auto& [depth, leaf_ver, script] : tuples) {
```

Reply:
Done.